web/modules/weather_blocks/src/Plugin/Block/Test/WeatherStoryBlock.php.test

```diff
@@ -112,7 +112,7 @@ final class WeatherStoryBlockTest extends Base
             "image" => null,
             "updated" => [
                 "formatted" => "Jun 24, 1977, 7:16 AM",
-                "utc" => "235984600",
+                "utc" => "1977-06-24T07:16:40+00:00",
```

Comment:
what's the backstory on this date? [Here](https://en.wikipedia.org/wiki/June_1977) it says the National Primary Drinking Water Regulations first went into effect, so I'm rolling with that for now.

Reply:
Let's go with that. I don't actually know. On the eAPD project, I put comments next to all the test dates to explain why I'd chosen them, but this one may just be totally arbitrary.